wwwroot/wave-generator.html

```diff
@@ -11,402 +11,403 @@
 </head>
 <body class="mdui-loaded mdui-appbar-with-toolbar mdui-drawer-body-left mdui-theme-primary-purple mdui-theme-accent-light-blue mdui-theme-layout-auto">
 
-<header class="mdui-appbar mdui-appbar-fixed">
-    <div class="mdui-toolbar mdui-color-theme">
-        <span class="mdui-btn mdui-btn-icon mdui-ripple" mdui-drawer="{target: '#main-drawer', swipe: true}">
-            <i class="mdui-icon material-icons">menu</i>
-        </span>
-        <a class="mdui-typo-headline" href="/Index" i18n-key="site-title-bar"></a>
-        <div class="mdui-toolbar-spacer"></div>
-        <span class="mdui-btn mdui-btn-icon mdui-ripple" mdui-menu="{target: '#language-menu'}">
-            <i class="mdui-icon material-icons">language</i>
-        </span>
-        <ul class="mdui-menu" id="language-menu">
-            <li class="mdui-menu-item">
-                <a class="mdui-ripple" i18n-key="site-lang-en" onclick="i18n.SetGlobalLocale('en')"></a>
-            </li>
-            <li class="mdui-menu-item">
-                <a class="mdui-ripple" i18n-key="site-lang-zh-CN" onclick="i18n.SetGlobalLocale('zh-CN')"></a>
-            </li>
-        </ul>
-    </div>
-</header>
-
-<div class="mdui-drawer" id="main-drawer">
-    <div class="mdui-list">
-        <a class="mdui-list-item mdui-ripple" href="/Index">Home</a>
-        <a class="mdui-list-item mdui-ripple" href="/Oscilloscope">Oscilloscope</a>
-        <a class="mdui-list-item mdui-ripple" href="/Supply">Power Source</a>
-        <a class="mdui-list-item mdui-ripple" href="wave-generator.html">Wave Gen</a>
-    </div>
-</div>
-
-<div class="mdui-container mdui-row" id="main-container">
-    <script type="text/javascript">
-        [1, 2].forEach((ch) => {
-            $("#main-container").append(`
-    <div class="mdui-col-xs-12 mdui-row" style="margin-top: 60px">
-        <div class="mdui-col-xs-12 mdui-row" style="display: flex">
-            <div class="mdui-col-xs-11">
-                <h1 class="mdui-typo-title">通道${ch}</h1>
-            </div>
-            <div class="mdui-col-xs-1" style="align-self: center; text-align: right">
-                <label class="mdui-switch">
-                    <input class="mdui-switch" id="wave-gen-${ch}-enable" type="checkbox"/>
-                    <i class="mdui-switch-icon"></i>
-                </label>
-            </div>
+    <header class="mdui-appbar mdui-appbar-fixed">
```

Comment:
The massive format changes here make it very difficult to determine whether the change is good or not. I'll take close look on it when available.